### PR TITLE
Implement password support for zipfldr

### PR DESCRIPTION
**Pull request by learn-more**

### CODE REVIEW

File: dll/shellext/zipfldr/CConfirmReplace.cpp [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+};
+
+
+eZipConfirmResponse _CZipAskReplace(HWND hDlg, PCSTR FullPath)
+{
+    PCSTR Filename = PathFindFileNameA(FullPath);
+    CConfirmReplace confirm(Filename);
+    INT_PTR Result = confirm.DoModal(hDlg);
+    switch (Result)
+    {
```
Comment (tambry) [inline dll/shellext/zipfldr/CConfirmReplace.cpp]: Should be indented per

File: dll/shellext/zipfldr/CConfirmReplace.cpp [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    case IDYES: return eYes;
```
Comment (tambry) [inline dll/shellext/zipfldr/CConfirmReplace.cpp]: Case labels should be idented [per the style guide](https://reactos.org/wiki/Coding_Style#Indentation).

File: dll/shellext/zipfldr/CZipPassword.cpp [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+eZipPasswordResponse _CZipAskPassword(HWND hDlg, const char* filename, CStringA& Password)
+{
+    if (filename)
+        filename = PathFindFileNameA(filename);
+    CZipPassword password(filename, &Password);
+    INT_PTR Result = password.DoModal(hDlg);
+    switch (Result)
+    {
+    case IDOK: return eAccept;
```
Comment (tambry) [inline dll/shellext/zipfldr/CZipPassword.cpp]: Same.

**Comment by yagoulas** [comment]
The case labels look good to me

**Pull request merged**